lit_gpt/adapter_v2.py

```diff
@@ -28,7 +28,7 @@
 class Config(BaseConfig):
     @property
     def mlp_class(self) -> Type:
-        return getattr(lit_gpt.adapter_v2, self._mlp_class)
+        return getattr(lit_gpt.adapter_v2, self.mlp_class_str)
```

Comment:
Why not `mlp_class_name`?
One character longer, but approx. 69% clearer what this arg stands for.